packages/forma-36-website/src/pages/guidelines/copy/grammar/capitalization/index.mdx

```diff
@@ -0,0 +1,24 @@
+# Capitalization
+
+## Use sentence style caps
+
+Sentence style caps, in which only the first word of a sentence or term is capitalized, should be used across the web app.
+This makes sentences a bit more scannable and our UX copy more approachable.
+Example uses:
+
+- labels
+- headings
+- titles
+- buttons
+- menu items
+
+![Build Status](/sentence-style-caps.jpg)
```

Comment:
Copy and paste fail. If we stick with images, we should also write better alt text in general. This should be something like `Example of correct usage of sentence style caps - use "View your website live" rather than "View Your Website Live"`